src/main/java/com/mongodb/Read.java

```diff
@@ -0,0 +1,70 @@
+package com.mongodb;
+
+import com.mongodb.client.*;
+import com.mongodb.diagnostics.logging.Loggers;
```

Comment:
This is an internal class and not expected to be used for public consumption.  Suggest just using the prefix string directly eg: "org.mongodb.driver"

Reply:
It's a bit contradictory to have a `public static final` and this comment just above: `This class is not part of the public API.`
Maybe this deserve a little patch?

Reply:
Yeah `Logger.getLogger("org.mongodb.driver").setLevel(Level.WARNING);` works.

The whole class is marked as non public - but due to sharing across the namespaces it has to be marked as public.  It does mean it could be moved or changed and that wouldn't be considered binary breaking,

Reply:
Done.